app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,18 @@
 <resources>
     <string name="app_name">Weather</string>
+    <string name="title_ok">OK</string>
+    <string name="text_o">o</string>
+    <string name="title_permission_denied">Permission denied</string>
+    <string name="title_permission_rationale">Location permission is needed for core functionality</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_permission_rationale` appears to be unused
 Unused resources make applications larger and slow down builds.